src/main/java/org/gridsuite/securityanalysis/server/service/SecurityAnalysisParametersService.java

```diff
@@ -0,0 +1,147 @@
+        SecurityAnalysisParameters.IncreasedViolationsParameters increasedViolationsParameters = new SecurityAnalysisParameters.IncreasedViolationsParameters();
+        increasedViolationsParameters.setFlowProportionalThreshold(flowProportionalThreshold);
+        increasedViolationsParameters.setLowVoltageAbsoluteThreshold(lowVoltageAbsoluteThreshold);
+        increasedViolationsParameters.setLowVoltageProportionalThreshold(lowVoltageProportionalThreshold);
+        increasedViolationsParameters.setHighVoltageAbsoluteThreshold(highVoltageAbsoluteThreshold);
+        increasedViolationsParameters.setHighVoltageProportionalThreshold(highVoltageProportionalThreshold);
+        return increasedViolationsParameters;
+    }
+
+    public static SecurityAnalysisParametersValues fromEntity(SecurityAnalysisParametersEntity entity) {
+        Objects.requireNonNull(entity);
+        return SecurityAnalysisParametersValues.builder()
+                .lowVoltageAbsoluteThreshold(entity.getLowVoltageAbsoluteThreshold())
+                .lowVoltageProportionalThreshold(entity.getLowVoltageProportionalThreshold())
+                .highVoltageAbsoluteThreshold(entity.getHighVoltageAbsoluteThreshold())
+                .highVoltageProportionalThreshold(entity.getHighVoltageProportionalThreshold())
+                .flowProportionalThreshold(entity.getFlowProportionalThreshold())
+                .build();
+    }
+
+    public static SecurityAnalysisParametersValues getDefaultSecurityAnalysisParametersValues() {
+        return SecurityAnalysisParametersValues.builder()
+                .lowVoltageAbsoluteThreshold(DEFAULT_LOW_VOLTAGE_ABSOLUTE_THRESHOLD)
+                .lowVoltageProportionalThreshold(DEFAULT_LOW_VOLTAGE_PROPORTIONAL_THRESHOLD)
+                .highVoltageAbsoluteThreshold(DEFAULT_HIGH_VOLTAGE_ABSOLUTE_THRESHOLD)
+                .highVoltageProportionalThreshold(DEFAULT_HIGH_VOLTAGE_PROPORTIONAL_THRESHOLD)
+                .flowProportionalThreshold(DEFAULT_FLOW_PROPORTIONAL_THRESHOLD)
+                .build();
+    }
+
+    public SecurityAnalysisParametersValues getParameters(UUID parametersUuid) {
+        var params = securityAnalysisParametersRepository.findById(parametersUuid).orElseThrow(() -> new SecurityAnalysisException(PARAMETERS_NOT_FOUND));
+        return SecurityAnalysisParametersService.fromEntity(params);
+    }
+
+    public UUID createParameters(SecurityAnalysisParametersValues securityAnalysisParametersValues) {
+        return securityAnalysisParametersRepository.save(securityAnalysisParametersValues.toEntity()).getId();
+    }
+
+    public UUID createDefaultParameters() {
+        return securityAnalysisParametersRepository.save(getDefaultSecurityAnalysisParametersValues().toEntity()).getId();
+    }
+
+    public Optional<UUID> createParameters(UUID sourceParametersId) {
```

Comment:
Because we can fail to duplicate the parameters, so we don't have an UUID to return. We return an optional and handle it in the controller to know what http response to give.